packages/dx-react-demos/src/demo-sources/.eslintrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "rules": {
-    "react/prop-types": "off"
+    "react/prop-types": "off",
+    "jsx-quotes": "off"
```

Comment:
Why do you switch off this rule?

Reply:
Because in `demo-source-data.json` I use next sentence `"openWrapperTag": "div className='card'",`

Reply:
Change it to `"openWrapperTag": "div className=\"card\""`